src/main/java/it/gov/pagopa/pu/debtpositions/repository/view/receipt/ReceiptDetailNoPIIViewRepository.java

```diff
@@ -17,7 +17,7 @@ public interface ReceiptDetailNoPIIViewRepository extends Repository<ReceiptDeta
       + "i.iuv as iuv, "
       + "r.paymentAmountCents as paymentAmountCents, "
       + "i.remittanceInformation as remittanceInformation, "
-      + "dp.description as debtPositionDescription, "
+      + "dpto.description as debtPositionDescription, "
```

Comment:
are you sure? if we want the debtPositionTypeDescription, we should also rename the DTO field
```suggestion
      + "dpto.description as debtPositionTypeDescription, "
```